CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+This document provides guidelines for contributing to this repository.
+
+## Development Environment Setup
+
+### Prerequisites
+* Basic tools: `curl`, `jq`, `oc`.
+* go: v1.20
+* Container engine: either `docker` or `podman`.
+
+### Unit Tests
+Go unit tests are included in various packages within the repository.
+Go unit tests are located within the tests directory, with filenames ending with _tests.go and with .go.
+
+### Running the Unit Tests Locally
+1. Clone your fork of the project.
+2. Navigate to the project's root directory
+3. To run all the Go unit tests in the repository, execute the following command `go test ./...`
+
+```
+?   	github.com/redhat-appstudio/segment-bridge.git/cmd/querygen	[no test files]
+ok  	github.com/redhat-appstudio/segment-bridge.git/querygen	0.002s
+ok  	github.com/redhat-appstudio/segment-bridge.git/queryprint	(cached)
+ok  	github.com/redhat-appstudio/segment-bridge.git/scripts	0.002s
+ok  	github.com/redhat-appstudio/segment-bridge.git/segment	(cached)
+ok  	github.com/redhat-appstudio/segment-bridge.git/stats	(cached)
+ok  	github.com/redhat-appstudio/segment-bridge.git/webfixture	(cached)
+```
+4. If you want to run tests for a specific package, you can do so by providing the package path, like this:
+```
+go test github.com/redhat-appstudio/segment-bridge.git/querygen
+```
+
+### Automated Tests
+[TBD]
+
+### Running the Automated Tests
+[TBD]
+
+### Test Coverage
+[TBD]
+
+### Commit Messages
+We use [gitlint](https://jorisroovers.com/gitlint/) to standardize commit messages, following the [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
+
+If you include a Jira ticket identifier (e.g., STONEO11Y-123) in the commit message, PR name, or branch name, it will link to the Jira ticket.
```

Comment:
```suggestion
If you include a Jira ticket identifier (e.g., RHTAPWATCH-123) in the commit message, PR name, or branch name, it will link to the Jira ticket.
```